apps/chat/src/components/Chatbar/ChatbarSettings.tsx

```diff
@@ -108,6 +109,7 @@ export const ChatbarSettings = () => {
         Icon: IconSquareOff,
         onClick: () => {
           dispatch(ConversationsActions.resetChosenConversations());
+          dispatch(ConversationsActions.resetChosenEmptyFolders());
```

Comment:
Why do we need separate action for that? We can do it into `resetChosenConversations`

Reply:
I think it's better to implement separate actions for improved separation of concerns, testability, and reusability.

Reply:
Code chsnged as suggested.